docs/Getting-Started.md

```diff
@@ -179,6 +179,40 @@ To solve this Fastify offers the `decorate` API, which adds custom objects to th
 
 To dig deeper into how Fastify plugins work, how to develop new plugins, and for details on how to use the whole Fastify API to deal with the complexity of asynchronously bootstrapping an application, read [the hitchhiker's guide to plugins](https://github.com/fastify/fastify/blob/master/docs/Plugins-Guide.md).
 
+<a name="plugin-loading-order"></a>
+### Loading order of your plugins
+To guarantee a consistent and predictable behavior of your application, we highly recommend to always load your code as showed below:
+```
+└── plugins (from the Fastify ecosystem)
+└── your plugins (your custom plugins)
+└── decorators
+└── hooks and middlewares
+└── your services
+```
+In this way you will always access all the properties declared in the current scope.<br/>
+As you know Fastify offers a solid encapsulation model, to help you build you application in single and independent services. If you want to register a plugin only for a subset of routes, you have just to replicate the above structure.
```

Comment:
"As you know Fastify offers" => "As discussed previously, Fastify offers"

"to help you build you application in" => "to help you build your application as"